telegram/message.py

```diff
@@ -168,6 +169,7 @@ def __init__(self,
         self.location = location
         self.venue = venue
         self.new_chat_member = new_chat_member
```

Comment:
I think we should help our users by  storing the value on `self._new_chat_member` and make `new_chat_member` a property + deprecation warning.